lib/src/components/NxTextInput/NxTextInput.scss

```diff
@@ -112,7 +116,7 @@
   &.disabled {
     .nx-text-input__input {
       &::placeholder {
-        color: var(--nx-color-interactive-border-disabled);
+        color: var(--nx-color-text-disabled);
```

Comment:
actually I think this is a mistaken duplicate from when I did NxTextInput. You should be able to remove it

Reply:
> actually I think this is a mistaken duplicate from when I did NxTextInput. You should be able to remove it

Fixed here! 21c3b0680a80d9ee52e1610f2b55bbeffe0db407